src/coreclr/jit/codegenarm64.cpp

```diff
@@ -2954,6 +2954,20 @@ void CodeGen::genCodeForStoreLclVar(GenTreeLclVar* lclNode)
         }
 #endif // FEATURE_SIMD
 
+#ifdef FEATURE_MASKED_HW_INTRINSICS
+        if ((compiler->lvaFfrRegister == lclNode->GetLclNum()) && (targetReg == REG_NA))
+        {
+            assert(targetType == TYP_MASK);
+            // We are about to store the FFR on stack. So first read the FFR register and
+            // store it in op1Reg. After that, op1Reg will be stored on the stack.
+            assert(varDsc->lvImplicitlyReferenced == 1);
+
+            regNumber op1Reg = data->GetRegNum();
+            assert(op1Reg != REG_NA);
+            GetEmitter()->emitIns_R(INS_sve_rdffr, EA_SCALABLE, op1Reg);
+        }
+#endif
+
```

Comment:
I don't really understand why we need to change codegen for `STORE_LCL_VAR` like this. Shouldn't the `GT_PHYSREG` with `REG_FFR` be what emits `sve_rdffr`?

Reply:
yes, perhaps i can do that. let me see.

Reply:
fixed.